### PR TITLE
task1

### PR DESCRIPTION
**Pull request by maxNia**
Снимок экрана 2021-06-23 в 18 47 57
try to upload some fields
Снимок экрана 2021-06-23 в 18 48 08
after update the second field from 'nikita' to 'NiKITA'

### CODE REVIEW

File: src/list/list.model.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import sequelize from "../config/database";
+
+interface ListInstance extends Model {
+  uuid: string;
+  name: string;
+}
+
+const List = sequelize.define<ListInstance>('lists', {
+  uuid: {
+    type: 'VARCHAR(36)',
```
Comment (CatDidenko) [inline src/list/list.model.ts:11]: should be uuiv4 type and autogenerated

File: src/contact/contact.services.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    
+    await Contact.create({uuid, name, email}); 
+  } catch (error: any) {
+    console.error(error.message);
+  }
+}
+
+export async function updateContactById(body: IUpdateContact, id: string) {
+  try {
+    await Contact.update(body, {
```
Comment (CatDidenko) [inline src/contact/contact.services.ts]: I suggested not to use models for querying in this task, but write SQL queries with the help of sequelize.query

File: src/contact/contact.services.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import {v4 as uuidv4} from 'uuid';
+import { IAddContact, IUpdateContact } from './contact.interfaces';
+import Contact from './contact.model';
+
+export async function addContact(body: IAddContact) {
+  try {
+    const {name, email} = body;
+    const uuid = uuidv4();
+    
+    await Contact.create({uuid, name, email});
```
Comment (CatDidenko) [inline src/contact/contact.services.ts]: I suggested not to use models for querying in this task, but write SQL queries with the help of sequelize.query

**Review by CatDidenko — CHANGES_REQUESTED** [review]

File: src/list/list.model.ts (hunk 1 above)
Comment (maxNia) [reply src/list/list.model.ts:11]: updated

File: src/contact/contact.services.ts (hunk 3 above)
Comment (maxNia) [reply src/contact/contact.services.ts]: updated

File: src/contact/contact.services.ts (hunk 2 above)
Comment (maxNia) [reply src/contact/contact.services.ts]: updated

**Review by CatDidenko — APPROVED** [review]

**Pull request merged**